torchmetrics/functional/text/wer.py

```diff
@@ -0,0 +1,63 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import List, Tuple, Union
+
+from torchmetrics.utilities.imports import _JIWER_AVAILABLE
+
+if _JIWER_AVAILABLE:
+    from jiwer import compute_measures
+
+
+def wer(
+    references: Union[str, List[str]],
+    predictions: Union[str, List[str]],
+    concatenate_texts: bool = False,
+) -> Union[float, Tuple[float, int, int]]:
```

Comment:
This does not match the output type of the `compute` in the modular interface. I assume this is where it is wrong
```suggestion
) -> float:
```